tests/pyspark_matrix_similarity_test.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from lib.pyspark_matrix_similarity import cosine_similarity_for_row
+from math import isclose
+from pyspark.ml.linalg import DenseVector
+
+from lib.pyspark_startup import init
+
+import findspark
```

Comment:
Próbowałeś odpalić ten plik testowy osobno? Jak robię !pytest tests/pyspark_matrix_similarity_test.py to się wysypuje, a przy !pytest tests/ wszystko jest ok. Chyba jakiś problem z importami

Reply:
Dziwne, u mnie działa. Działa każdy z osobna i wszystkie razem

Reply:
spark.init() przeniesiony przed klasę.